### ISSUE TITLE
Fix syntax issues

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#244]
```
Unused imports in oauth2client/appengine.py:
import urllib
import urlparse
from urlparse import parse_qsl
from cgi import parse_qsl

In OAuth2Handler.get, the line continued after redirect_uri = 
util._add_query_parameter( should indent by 4, not 2 spaces.

Invalid import order in apiclient/discovery.py:
Should be
import copy
from email.mime.multipart import MIMEMultipart
from email.mime.nonmultipart import MIMENonMultipart
import keyword
import logging
import mimetypes
import os
import re
import urllib
import urlparse

from apiclient.errors import HttpError
from apiclient.errors import InvalidJsonError
from apiclient.errors import MediaUploadSizeError
from apiclient.errors import UnacceptableMimeTypeError
from apiclient.errors import UnknownApiNameOrVersion
from apiclient.errors import UnknownFileType
from apiclient.http import HttpRequest
from apiclient.http import MediaFileUpload
from apiclient.http import MediaUpload
from apiclient.model import JsonModel
from apiclient.model import MediaModel
from apiclient.model import RawModel
from apiclient.schema import Schemas
import httplib2
import mimeparse
from oauth2client.anyjson import simplejson
from oauth2client.util import _add_query_parameter
from oauth2client.util import positional
import uritemplate

MEDIA_BODY_PARAMETER_DEFAULT_VALUE keys should indent by 4, not 2 spaces

apiclient/tests/test_errors.py:
Should be a space after colon in {'status':'400'} in line
resp, content = fake_response('}NOT OK', {'status':'400'}, reason=None)

apiclient/tests/test_oauth2client.py:
In BasicCredentialsTests.test_token_refresh_success, 
http = HttpMockSequence([
should indent by 4 instead of 2 in the continued lines.

apiclient/tests/test_oauth2client_appengine.py:
In DecoratorTests.test_required, the indent after "if 
self.decorator._token_response_param:" should have two spaces, not four.
```

Original issue reported on code.google.com by `dhermes@google.com` on 6 Mar 2013 at 4:58

**Comment by GoogleCodeExporter** [comment]
```
https://codereview.appspot.com/7516043/
```

Original comment by `dhermes@google.com` on 6 Mar 2013 at 5:16
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision a7ca6262a7b3.
```

Original comment by `dhermes@google.com` on 6 Mar 2013 at 5:39
- Changed state: **Fixed**
